app/views/layouts/application.html.erb

```diff
@@ -1,7 +1,7 @@
 <!DOCTYPE html>
 <html>
   <head>
-    <title>SampleApp</title>
+  <title><%= yield(:title) %> | Ruby on Rails Tutorial Sample App</title>
```

Comment:
` Ruby on Rails Tutorial Sample App`
Tìm hiểu i18n và sử dụng cho code trên

-> Chú thực hiện self check toàn bộ file html rồi thực hiện:
 + sử dụng i18n cho toàn bộ text
 + Dùng `link_to` tag thay vì `a` tag nhé